src/AppAny.Quartz.EntityFrameworkCore.Migrations.MySql/EntityTypeConfigurations/QuartzBlobTriggerEntityTypeConfiguration.cs

```diff
@@ -1,7 +1,7 @@
 using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Metadata.Builders;
 
-namespace AppAny.Quartz.EntityFrameworkCore.Migrations.MySql
+namespace AppAny.Quartz.EntityFrameworkCore.Migrations.MySql.EntityTypeConfigurations
```

Comment:
If you want `AppAny.Quartz.EntityFrameworkCore.Migrations.{{ Provider }}.EntityTypeConfigurations` namespace for `EntityTypeConfigurations` you will also need to change PostgreSQL and SQLServer

Reply:
Yeah I don't like it, I will change the namespaces for everything to just `AppAny.Quartz.EntityFrameworkCore.Migrations.{{ Provider }}`